presto-main/src/test/java/com/facebook/presto/operator/scalar/TestArrayFunctions.java

```diff
@@ -27,10 +27,8 @@ public class TestArrayFunctions
     @Test
     public void testArrayConstructor()
     {
-        tryEvaluateWithAll("array[" + Joiner.on(", ").join(nCopies(254, "rand()")) + "]", new ArrayType(DOUBLE));
-        assertNotSupported(
```

Comment:
Please put back the negative test with a value exceeding the new upper limit

Reply:
Hmm, I was considering doing recursive concatenation so there would be no upper limit in theory beyond the computing limitations, wdyt about that?

Reply:
Probably not a good idea since it could lead to excessive memory consumption and GC storms as a result. Maybe @kaikalur can chime in on whether the new limits will suffice

Reply:
I will add the negative test case back for now. Can you please help check what might be the problem with two of the checks failing, have not encountered them before and I only added the changes suggested and fixed the styles. Thanks!

Reply:
Looks like both failures are due to flaky tests. I filed an issue for [one](https://github.com/prestodb/presto/issues/23006), and there already exists an [issue](https://github.com/prestodb/presto/issues/22848) for the other. 
If these tests continue to block you, feel free to disable them

Reply:
Hi, how would disable the failing tests